packages/storybook-html/src/components/core/typography.story.mdx

```diff
@@ -0,0 +1,192 @@
+
+## Size
+
+We use 16px as the base or deafult for most browsers and devices.
+
+<table>
+  <thead>
+    <tr>
+      <th>Rem</th>
+      <th>Px</th>
+      <th>SASS Variable</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>0.75</td>
+      <td>12</td>
+      <td>$font-size-xs</td>
+    </tr>
+    <tr>
+      <td>0.875</td>
+      <td>14</td>
+      <td>$font-size-s</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>16</td>
+      <td>$font-size-m</td>
+    </tr>
+    <tr>
+      <td>1.125</td>
+      <td>18</td>
+      <td>$font-size-l</td>
+    </tr>
+    <tr>
+      <td>1.25</td>
+      <td>20</td>
+      <td>$font-size-xl</td>
+    </tr>
+    <tr>
+      <td>1.5</td>
+      <td>24</td>
+      <td>$font-size-xxl</td>
+    </tr>
+    <tr>
+      <td>2</td>
+      <td>32</td>
+      <td>$font-size-xxxl</td>
+    </tr>
+    <tr>
+      <td>2.5</td>
+      <td>40</td>
+      <td>$font-size-xxxxl</td>
+    </tr>
+    <tr>
+      <td>3</td>
+      <td>48</td>
+      <td>$font-size-xxxxxl</td>
+    </tr>
+  </tbody>
+</table>
+
+## Embedding in projects
+
+### Font licence
+
+You can use them freely in your products & projects - print or digital, commercial or otherwise. However, you can't sell the fonts on their own.
+
+<table>
+  <thead>
+    <tr>
+    <th>Font</th>
+    <th>License</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Roboto</td>
+      <td>Apache License 2.0<br /> 
+      <a href="https://fonts.google.com/specimen/Roboto#license">Google, Roboto License</a></td>
+    </tr>
+    <tr>
+      <td>Merriweather</td>
+      <td>SIL Open Font License (OFL)<br />
+      <a href="https://fonts.google.com/specimen/Merriweather#license">Google, Merriweather License</a></td>
+    </tr>
+    <tr>
+      <td>Montserrat</td>
+      <td>SIL Open Font License (OFL)<br />
+      <a href="https://fonts.google.com/specimen/Montserrat#license">Google, Montserrat License</a></td>
+    </tr>
+  </tbody>
+</table>
+
+### Font stacks
+
+The following font family declarations use widely available fonts.
+A user's web browser will look at the comma-separated list and will
+attempt to use each font in turn until it finds one that is available
+on the user's computer. The final "generic" font (sans-serif, serif or
+monospace) hints at what type of font to use if the web browser doesn't
+find any of the fonts in the list.
+
+Roboto
+```css
+font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
+```
+Merriweather
+```css
+font-family: Merriweather, Georgia, serif;
+```
+Montserrat
+```css
+font-family: Montserrat, "Helvetica Neue", Helvetica, Arial, sans-serif;
+```
+
+Monospace (Based on available system fonts across common OS's and devices)
+```css
+font-family: Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace;
```

Comment:
Do we need all these fonts.. lots in them ?

Reply:
I don't think so but until i can research more that was the recommendation for a modern stack. I assume it is due to the range of devices.

Reply:
Ok, thats fine then